Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:9.10.1-alpine as builder
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh alpine-sdk python python-dev py-pip build-base
+RUN npm install -g yarn
+ADD package.json .
```

Comment:
lockfile is missing

Reply:
Are you referring to `package-lock.json`? it is not in the repository. If you are referring to the yarn lock file, it is not necessary at that stage since the objective is only to install the npm dependencies and cache these in a Docker layer.

Reply:
I am referring to `yarn.lock`. 

> it is not necessary at that stage since the objective is only to install the npm dependencies and cache these in a Docker layer

This is exactly why it's needed. Without this when you install npm dependencies you can't be sure what will be installed. We ran it in travis as well https://github.com/Neufund/TypeChain/blob/master/.travis.yml#L10